recipes/essentials/all/conanfile.py

```diff
@@ -0,0 +1,102 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import export_conandata_patches, get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+
+required_conan_version = ">=2.0"
+
+class EssentialsConan(ConanFile):
+    name = "essentials"
+    description = "essentials is a small c++ library that offers very basic capabilities for applications and libraries."
+    license = "MIT License"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://essentials.seadex.de/"
+    topics = ("utility", "C++", "library")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_unit_tests": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_unit_tests": False,
+        "spdlog/*:header_only": True
+    }
+
+    requires = "spdlog/1.11.0", "fmt/9.1.0"
+    build_policy = "missing"
```

Comment:
build_policy is not allowed in conancenter recipes, only usage requirements are allowed